src/core/server/app/middleware/graphql/apolloServer.ts

```diff
@@ -31,6 +32,7 @@ function contextProvider(options: ContextProviderOptions) {
       tenant,
       logger,
       site,
+      traceID: uuidv4(),
```

Comment:
The `id` on this already is sent as the Trace ID in the `X-Trace-ID` header of the response.